src/main/java/com/eva/model/person/applicant/PossibleApplicationStatus.java

```diff
@@ -13,4 +13,22 @@
         this.status = status;
     }
 
+    public static PossibleApplicationStatus getStatus(String status) {
+        if (status.equals(RECEIVED.status)) {
+            return RECEIVED;
+        } else if (status.equals(PROCESSING.status)) {
+            return PROCESSING;
+        } else if (status.equals(ACCEPTED.status)) {
+            return ACCEPTED;
+        } else if (status.equals(REJECTED.status)) {
+            return REJECTED;
+        } else {
+            throw new IllegalArgumentException();
+        }
+    }
```

Comment:
with relation to my previous comment, this can be removed i think but im not sure how the constructors and the enums tagged to strings work.